lib/generators/scim_rails/templates/initializer.rb

```diff
@@ -18,6 +18,10 @@
   # authenticatable model.
   config.scim_users_scope = :users
 
+  # Determine whether the create endpoint updates users that already exist
+  # or throws an error (returning 409 Conflict in accordance with SCIM spec)
+  config.scim_user_prevent_update_on_create = :false
```

Comment:
Can you help me understand the use of the symbol `:false` here? Why not use the boolean value `false`?

Reply:
Good catch @armilam this file is not used by the gem but is a guide for users to add to their projects. The way it's being used this would actually result in boolean true `if ScimRails.config.scim_user_prevent_update_on_create` which would not be what we want. I agree this should be a boolean false as the example. On the plus side I tested if this was `nil` and it did keep the old behavior

Reply:
We should use `config.scim_user_prevent_update_on_create = false` here as `:false` will evaluate to true in the conditional

Reply:
This was an oversight on my part, thank you for catching it. Fixed now.